<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui_Main</class>
 <widget class="QMainWindow" name="Gui_Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KetoEvent Logger</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/keto_48</normaloff>:/icons/keto_48</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="comboTeams">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>Select team from the list</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTimeEdit" name="timeFinish">
      <property name="toolTip">
       <string>Enter team's finish time here</string>
      </property>
      <property name="currentSection">
       <enum>QDateTimeEdit::HourSection</enum>
      </property>
      <property name="displayFormat">
       <string>HH:mm</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="taskList">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>Log team's accomplished tasks here</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="resizeMode">
       <enum>QListView::Adjust</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="findTaskLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="findTaskEdit">
        <property name="toolTip">
         <string>Enter task's name and press ENTER</string>
        </property>
        <property name="placeholderText">
         <string>Find task by name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="clearButton">
        <property name="toolTip">
         <string>Clear task name</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/clear_22</normaloff>:/icons/clear_22</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionEvents"/>
   <addaction name="actionTeams"/>
   <addaction name="actionTasks"/>
   <addaction name="actionCombos"/>
   <addaction name="actionRankings"/>
   <addaction name="actionSettings"/>
   <addaction name="actionAbout"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionQuickAdd"/>
   <addaction name="actionSort"/>
   <addaction name="actionCombine"/>
  </widget>
  <action name="actionTeams">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/teams_48</normaloff>:/icons/teams_48</iconset>
   </property>
   <property name="text">
    <string>Teams</string>
   </property>
   <property name="toolTip">
    <string>Edit teams</string>
   </property>
  </action>
  <action name="actionTasks">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/tasks_48</normaloff>:/icons/tasks_48</iconset>
   </property>
   <property name="text">
    <string>Tasks</string>
   </property>
   <property name="toolTip">
    <string>Edit tasks</string>
   </property>
  </action>
  <action name="actionRankings">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/rankings_48</normaloff>:/icons/rankings_48</iconset>
   </property>
   <property name="text">
    <string>Rankings</string>
   </property>
   <property name="toolTip">
    <string>View rankings</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/settings_48</normaloff>:/icons/settings_48</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Change settings</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/about_48</normaloff>:/icons/about_48</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About applet</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/exit_48</normaloff>:/icons/exit_48</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit applet</string>
   </property>
  </action>
  <action name="actionEvents">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/events_48</normaloff>:/icons/events_48</iconset>
   </property>
   <property name="text">
    <string>Events</string>
   </property>
  </action>
  <action name="actionCombos">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/combos_48</normaloff>:/icons/combos_48</iconset>
   </property>
   <property name="text">
    <string>Combos</string>
   </property>
  </action>
  <action name="actionConsole">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/console_48</normaloff>:/icons/console_48</iconset>
   </property>
   <property name="text">
    <string>Console</string>
   </property>
   <property name="toolTip">
    <string>Show console</string>
   </property>
  </action>
  <action name="actionLogTime">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/log_16</normaloff>:/icons/log_16</iconset>
   </property>
   <property name="text">
    <string> Log</string>
   </property>
  </action>
  <action name="actionSort">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/tasks_sort_16</normaloff>:/icons/tasks_sort_16</iconset>
   </property>
   <property name="text">
    <string>Sort logs</string>
   </property>
  </action>
  <action name="actionCombine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/combos_48</normaloff>:/icons/combos_48</iconset>
   </property>
   <property name="text">
    <string>Combine</string>
   </property>
  </action>
  <action name="actionLockTeam">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/locked_16</normaloff>:/icons/locked_16</iconset>
   </property>
   <property name="text">
    <string>Lock team</string>
   </property>
  </action>
  <action name="actionQuickAdd">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/team_add_16</normaloff>:/icons/team_add_16</iconset>
   </property>
   <property name="text">
    <string>Quick add</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
